Docs/source/running_cpp/parameters.rst

```diff
@@ -922,24 +922,19 @@ Boundary conditions
 Diagnostics and output
 ----------------------
 
-* ``amr.plot_int`` (`integer`)
-    The number of PIC cycles inbetween two consecutive data dumps. Use a
-    negative number to disable data dumping.
-
-* ``warpx.dump_plotfiles`` (`0` or `1`) optional
-    Whether to dump the simulation data in
-    `AMReX plotfile <https://amrex-codes.github.io/amrex/docs_html/IO.html>`__
-    format. This is ``1`` by default, unless WarpX is compiled with openPMD support.
-
-* ``warpx.dump_openpmd`` (`0` or `1`) optional
-    Whether to dump the simulation data in
-    `openPMD <https://github.com/openPMD>`__ format.
-    When WarpX is compiled with openPMD support, this is ``1`` by default.
-
-* ``warpx.openpmd_backend`` (``h5``, ``bp`` or ``json``) optional
-    I/O backend for
-    `openPMD <https://github.com/openPMD>`__ dumps.
-    When WarpX is compiled with openPMD support, this is ``h5`` by default.
+* ``amr.plot_int`` (`integer`) optional
+    The number of PIC cycles (interval) in between two consecutive `plotfile` data dumps.
+    Use a negative number to disable data dumping.
+    This is ``-1`` (disabled) by default.
+
+* ``warpx.openpmd_int`` (`integer`) optional
+    The number of PIC cycles (interval) in between two consecutive `openPMD <https://github.com/openPMD>`__ data dumps.
+    Requires to build WarpX with ``USE_OPENPMD=TRUE`` (uses `openPMD-api <https://github.com/openPMD/openPMD-api>`__).
```

Comment:
Instead of a link to `openPMD-api`, could we put a link to the documentation on how to compile with openPMD support: something like
```
Requires to build WarpX with openPMD support (see instructions here: https://warpx.readthedocs.io/en/latest/building/openpmd.html)
```

Reply:
Yes, for such links to work we generally have to add anchor links all over our sphinx docs.